dashboard/app/helpers/openai_chat_helper.rb

```diff
@@ -10,7 +10,9 @@ def initialize(api_key, model)
       @model = model
     end
 
-    def request_chat_completion(messages, temperature = DEFAULT_TEMPERATURE)
+    # Extra empty set is included to provide generic coverage to new parameters that OpenAI adds in the future
+    # Examples include response_format for json response formatting and tools for function calling
+    def request_chat_completion(messages, temperature = DEFAULT_TEMPERATURE, extra: {})
```

Comment:
Could this generic `extra` param just be an optional `request_parameters` so you don't have to `merge` later on?